### ISSUE TITLE
Flame Leviathan shutdown (10m)

### ISSUE THREAD

**Issue by Godmatik** [GameFreedomGG/Sindragosa#393]
If you only kill one of the turrets on Flame Leviathan you can cause a shutdown in 10 man to stun the boss (not certain about 25m, you should probably check that also).

How it should work: you need to kill all turrets to start a shutdown (two in 10m, four in 25m).

**Comment by Godmatik** [comment]
Just an update on this, from what I am being told, as long as you kill the front turrets (1 in 10m, 2 in 25m), it will allow you to press the switch to shutdown the boss

**Comment by UltraNix** [comment]
Cannot reproduce it.

**Comment by Godmatik** [comment]
Apparently it only appears "sometimes" when you kill 2 turrets.

@UltraNix 

See below:

https://imgur.com/a/hvs3rWq

**Comment by UltraNix** [comment]
@Godmatik Hmm, but you activated shutdown by clicking? It should be done by killing turrets, not clicking anything.

**Comment by Godmatik** [comment]
correct, as you see in that link, there was something you could 'click' after two turrets die, to trigger shutdown. An interactable object.

**Comment by Soufly** [comment]
It doesnt even activate the shutdown after we killed the turrets... Nice ine
